.github/workflows/ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       - name: Install FuseML
         run: |
           ./dist/fuseml-installer version
-          make fuseml-install-with-mlflow
+          make fuseml-install-with-extensions
 
       - name: Install KFServing
```

Comment:
At this point you can also delete this step.

Reply:
Right, I missed that one